langserve/server.py

```diff
@@ -116,8 +216,10 @@ def add_routes(
     output_type: Union[Type, Literal["auto"], BaseModel] = "auto",
     config_keys: Sequence[str] = ("configurable",),
     include_callback_events: bool = False,
-    enable_feedback_endpoint: bool = False,
     per_req_config_modifier: Optional[PerRequestConfigModifier] = None,
+    enable_feedback_endpoint: bool = False,
```

Comment:
Should we get rid of this and see who complains? I feel like no one is probably using this outside of people we've actually pointed here, and it make the API gross to have it along with enabled/disabled endpoints...

I feel like we're really early on for backwards compat issues on langserve. Ideally we could do like minor version bumps or something and make small changes on compat